src/documentation/runners/mapreduce.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: default
+title: "Apache Hadoop MapReduce Runner"
+permalink: /documentation/runners/mapreduce/
+redirect_from: /learn/runners/mapreduce/
+---
+# Using the Apache Hadoop MapReduce Runner
+
+The Apache Hadoop MapReduce Runner can be used to execute Beam pipelines using [Apache Hadoop](http://hadoop.apache.org/).
+
+The [Beam Capability Matrix]({{ site.baseurl }}/documentation/runners/capability-matrix/) documents the currently supported capabilities of the Apache Hadoop MapReduce Runner.
+
+## Apache Hadoop MapReduce Runner prerequisites and setup
+You need to have an Apache Hadoop environment with either [Single Node Setup](https://hadoop.apache.org/docs/r1.2.1/single_node_setup.html) or [Cluster Setup](https://hadoop.apache.org/docs/r1.2.1/cluster_setup.html)
+
+The Apache Hadoop MapReduce runner currently supports Apache Hadoop 2.8.1 version.
+
+You can add a dependency on the latest version of the Apache Hadoop MapReduce runner by adding to your pom.xml the following:
+```java
+<dependency>
+  <groupId>org.apache.beam</groupId>
+  <artifactId>beam-runners-mapreduce</artifactId>
+  <version>{{ site.release_latest }}</version>
+</dependency>
+```
+
+## Deploying Apache Hadoop MapReduce with your application
+To execute in a local hadoop environment, use this command:
```

Comment:
capitalize Hadoop (multiple places on the page)

Reply:
done